txnkv/txnsnapshot/snapshot.go

```diff
@@ -387,6 +388,7 @@ func (s *KVSnapshot) batchGetSingleRegion(bo *retry.Backoffer, batch batchKeys,
 
 	pending := batch.keys
 	var resolvingRecordToken *int
+	firstTime := true
```

Comment:
Name it `UseConfigurableKVTimeout` is more understandable and some comments are also needed.